pkgs/development/python-modules/faraday-agent-parameters-types/default.nix

```diff
@@ -28,7 +30,10 @@ buildPythonPackage rec {
       --replace '"pytest-runner",' ""
   '';
 
-  pythonImportsCheck = [ "faraday_agent_parameters_types" ];
```

Comment:
I'm not sure why, but even without adding the `packaging` dependency, the pythonImportsCheck phase succeeded. But testing manually confirmed there was an error in the package not detected by this method.

Reply:
Because the binary and the module are a bit different in regard of their runtime requirement. `import xyz` usually works without `packaging`.